flink-contrib/docker-flink/docker-entrypoint.sh

```diff
@@ -40,3 +39,6 @@ elif [ "$1" = "taskmanager" ]; then
 else
     $@
 fi
+
+# prevent script to exit
+tail -f /dev/null
```

Comment:
Why is this needed?

https://github.com/apache/flink/blob/master/flink-contrib/docker-flink/Dockerfile#L49 already prevents the container from exiting till the daemon dies.

This on the other hand will never let the container exit.